tests/test_ios_selected_test_execution.py

```
@@ -0,0 +1,99 @@
+
+    def test_rejects_missing_execution_summary_for_requested_filter(self) -> None:
+        result = self.run_guard("** TEST SUCCEEDED **", "cmuxUITests/testMissingMethod")
+        self.assertNotEqual(result.returncode, 0)
+        self.assertEqual(
+            result.stderr,
+            "selected iOS test execution summary was not found; "
+            "verify the test log format\n",
+        )
+
+    def test_missing_log_does_not_echo_its_path(self) -> None:
+        missing_log = "/tmp/private-selected-test-output.log"
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/private-selected-test-output.log"

(S108)